packages/daimo-contract/src/backcompat/entryPointV06.ts

```diff
@@ -0,0 +1,1013 @@
+export const entryPointV06Address =
+  "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
+export const entryPointV06ABI = [
```

Comment:
is there a corresponding codegen for `entryPointV07`?

Reply:
yes, `entryPointABI` in the codegen output

I've pasted the old one into the `backcompat/` folder so we don't have to pull in both the new and old version of the `account-abstraction` forge dependency